docs/about_sat/introduction.md

```diff
@@ -95,48 +100,40 @@ several steps that need to be executed from a management NCN.
 
 #### Man Pages - Interactive and Non-interactive Modes
 
-To view a `sat` man page from a Kubernetes manager node, use `sat-man` on the manager node as shown in the following
-example.
+To view a `sat` man page from a Kubernetes manager node, use `sat-man` on the manager node.
+
+(`ncn-m001#`) Here is an example:
 
 ```screen
-ncn-m001# sat-man status
+sat-man status
 ```
 
 A man page describing the SAT container environment is available on the Kubernetes manager nodes, which can be viewed
 either with `man sat` or man `sat-podman` from the manager node.
 
+(`ncn-m001#`) Here are examples:
+
 ```screen
-ncn-m001# man sat
+man sat
 ```
 
 ```screen
-ncn-m001# man sat-podman
+man sat-podman
 ```
 
 ## Command Prompt Conventions in SAT
 
-The host name in a command prompt indicates where the command must be run. The account that must run the command is
-also indicated in the prompt.
+The host name in a command prompt indicates where the command must be run. The
+user account that must run the command is also indicated in the prompt.
 
-- The `root` or super-user account always has the `#` character at the end of the prompt and has the host name of the
-  host in the prompt.
-- Any non-`root` account is indicated with account@hostname>. A user account that is neither `root` nor `crayadm` is
-  referred to as `user`.
-- The command prompt inside the SAT container environment is indicated with the string as follows. It also has the "#"
-  character at the end of the prompt.
+- The `root` or super-user account always has host name in the prompt and the
+  `#` character at the end of the prompt.
+- Any non-root account is indicated with `account@hostname`. A non-privileged
+  account is referred to as user.
+- The command prompt inside the SAT container environment is indicated with the
+  string as follows. It also has the `#` character at the end of the prompt.
 
 | Command Prompt | Meaning |
 |----------------|---------|
-| `ncn-m001#` | Run on one of the Kubernetes Manager servers. (**Non-interactive**) |
+| `ncn-m001#` | Run the command on one of the Kubernetes Manager servers. (**Non-interactive**) |
```

Comment:
If we want to match docs-csm more closely, we could actually change `ncn-m001#` to `ncn-m#`.

Reply:
@haasken-hpe, does this apply to every situation, or are there sometimes where users need to run commands on a specific node? Want to check before I go and change this everywhere.

Reply:
In theory, `sat` commands should work on any Kubernetes master (i.e. Control Plane) node. However, I do see some examples where we are assuming that we are running on `ncn-m001` specifically. For example,the installation instructions have the admin set up SAT configuration on `ncn-m001` and then copy it to other `ncn-m` nodes.

The easiest thing to do might actually be to change the "Meaning" column in this case to just state that this `ncn-m001#` means the command should be run on that specific host `ncn-m001`.  Here's how CSM shows that in their command prompt conventions, for example: https://github.com/Cray-HPE/docs-csm/blob/release/1.6/introduction/documentation_conventions.md?plain=1#L79

Reply:
Updated the meaning of `ncn-m001` in the key so that it matches CSM. This will also avoid any problems with SAT that might be caused by not running on this specific node.